model/src/cfd.rs

```diff
@@ -1435,6 +1432,17 @@ pub fn calculate_long_liquidation_price(leverage: Leverage, price: Price) -> Pri
     price * leverage / (leverage + 1)
 }
 
+/// calculates short liquidation price
+///
+/// Note: if leverage == 1, then the liquidation price will go towards infinity.
+/// This is represented as Price::INFINITE
+pub fn calculate_short_liquidation_price(leverage: Leverage, price: Price) -> Price {
+    if (leverage - 1) == 0 {
```

Comment:
Any reason why you are not using the `Leverage::ONE` constant? 
To make the division by `0` more obvious? I think it would still read better using the constant...

Reply:
`Leverage - Leverage` much sense because (imho) `Leverage-Leverage` should return `Leverage` but `Leverage::new` does not allow `0`, which means, a naive implementation of `impl Sub<Leverage> for Leverage` would not work.

Reply:
I am confused by your answer.
Why not this condition:

```suggestion
    if leverage == Leverage::ONE {
```

Reply:
ah, yea, that would work 😅